src/components/shared/TextareaField/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+type TextareaFieldProps = {
+  id: string;
+  name: string;
+  label?: string;
+  maxLength?: number;
+  onChange: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+  onBlur: () => void;
+  value: string;
+};
+
+const TextareaField = ({
```

Comment:
Should we capitalize the A in TextareaField? I see pros and cons

Reply:
Yeah, I was on the fence about this one. I went with `Textarea` because it's pretty common in the community to just type it as "textarea" (one word). After re-thinking it, `textarea` isn't recognized by W3 as one word, so I'll rename to `TextArea` since it is _technically_ two words.